### DISCUSSION TITLE
Can views and stored procedures share an entity?

### DISCUSSION THREAD

**Discussion by pholcroft** [Azure/data-api-builder#2113] [Q&A] [answered]
I have a Contact entity which retrieves data via a view. I also have  a ContactCreate stored procedure for create and update. Currently I have them as separate entities, but is there anyway to have POST/PUT on Contact use the StoredProcedure?

**Comment by sezal98** [comment, marked as answer]
Yes Stored procedures can have POST/PUT as http rest methods. We need to configure this in the config file appropriately else we would get 405 error code. 
Please refer to this. https://learn.microsoft.com/en-us/azure/data-api-builder/database-objects#stored-procedures
